storage/client/blob/src/main/java/com/azure/storage/blob/models/package-info.java

```diff
@@ -3,6 +3,6 @@
 // Code generated by Microsoft (R) AutoRest Code Generator.
 
 /**
- * Package containing the data models for AzureBlobStorage.
+ * This package contains the data models for AzureBlobStorage.
```

Comment:
I think there was a architectural change for the comment convention for package-info files